webpack/assets/javascripts/react_app/ReactApp/Context/ForemanContext.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
```

Comment:
I'm wondering about the right approach

1. Having one global context that will give you all the data you ever need (based on namespaces of curse)
2. Having multiple contexts and manage them by a domain/namespace level.

Correct me if I wrong but with this PR you choose approach 1?

Reply:
+1 to approach 1 which seems to be what this PR does.  But I don't think it's bad to have more specific uses of React context elsewhere in the app (for example I use it for the ForemanModal component to pass data to its subcomponents.)